### PR TITLE
Hansen/add course after signup

### PR DESCRIPTION
**Pull request by hlillemark**
https://www.notion.so/User-Groups-Add-course-after-sign-up-598a68822ff44fa2a918f0c31301d4be

### CODE REVIEW

File: stanford/quiz/views.py [hunk 1]
```diff
@@ -145,6 +146,12 @@ class StudentCourseViewSet(ModelViewSet):
     def get_queryset(self):
         return Course.objects.filter(is_active=True, students__in=[self.request.user.student])
 
+class PublicCourseViewSet(ModelViewSet):
+    serializer_class = StudentCourseSerializer
+
+    def get_queryset(self):
+        return Course.objects.filter(private = False)
```
Comment (saahilchadha1) [inline stanford/quiz/views.py:153]: this is where you can add another condition tha the course has `is_active` true

File: stanford/quiz/views.py [hunk 2]
```diff
@@ -751,6 +758,35 @@ def send_whatsapp_view(request):
...
+
+@login_required
+def remove_course(request):
+    """ Removes user from a course"""
+    if request.method == 'POST':
+        access_code = request.POST.get('access_code').strip()
+        student = request.user.student
+        if student and access_code != "":
+            if not Course.objects.filter(code=access_code).filter(students=student):
+                return redirect('dashboard') # TODO how to display error easily?
```
Comment (saahilchadha1) [inline stanford/quiz/views.py:783]: when would this error be reached? aren't they only able to remove courses they're enrolled in?

**Review by saahilchadha1 — CHANGES_REQUESTED** [review]
modify this so it only displays active public courses in the table

File: stanford/quiz/views.py (hunk 2 above)
Comment (hlillemark) [reply stanford/quiz/views.py:783]: fragment from old way of deleting courses through course number, can just leave it in if we re implement that functionality

**Review by saahilchadha1 — APPROVED** [review]
LGTM

**Pull request merged**